e2e/content-services/tag-component.e2e.ts

```diff
@@ -68,12 +68,13 @@ describe('Tag component', () => {
 
         await apiService.getInstance().core.tagsApi.addTag(nodeId, tags);
 
-        await loginPage.login(acsUser.email, acsUser.password);
+        await loginPage.login(browser.params.testConfig.admin.email, browser.params.testConfig.admin.password);
```

Comment:
Any reason why we execute the tests with admin user instead of the user that created the file?

Reply:
I can check

Reply:
done